scripts/repo-sync.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly HELM_URL=https://storage.googleapis.com/kubernetes-helm
+readonly HELM_TARBALL=helm-v2.12.2-linux-amd64.tar.gz
+readonly STABLE_REPO_URL=https://charts.reactiveops.com/stable/
+readonly INCUBATOR_REPO_URL=https://charts.reactiveops.com/incubator/
+readonly S3_BUCKET_STABLE=s3://charts.reactiveops.com/stable
+readonly S3_BUCKET_INCUBATOR=gs://charts.reactiveops.com/incubator
+
+main() {
+    setup_helm_client
```

Comment:
Helm is already installed in the helmpack chart-testing image. Do we need to install a specific version again here? Not a big deal either way.

Reply:
Yeah, I had that thought as well.  I think it's worth keeping in case we want to update it to a newer version.  This was also done in the main helm repo, and I think this is why.